kong/pdk/log.lua

```diff
@@ -809,6 +817,7 @@ do
 
       local root = {
         request = {
+          id = request_id.get() or "",
```

Comment:
Could you please cache `request_id.get` at the module level? I think `kong.log.serialize()` is a hot path.